Regression due to `_from_sequence`

Original: https://github.com/dask/dask/issues/10601 cc @jrbourbeau 

```
_______________________________ test_timestamp96 _______________________________
[gw1] linux -- Python 3.10.12 /usr/share/miniconda3/envs/test-environment/bin/python3.10

tmpdir = local('/tmp/pytest-of-runner/pytest-0/popen-gw1/test_timestamp960')

    @FASTPARQUET_MARK
    def test_timestamp96(tmpdir):
        fn = str(tmpdir)
        df = pd.DataFrame({"a": [pd.to_datetime("now", utc=True)]})
        ddf = dd.from_pandas(df, 1)
        ddf.to_parquet(fn, engine="fastparquet", write_index=False, times="int96")
        pf = fastparquet.ParquetFile(fn)
        assert pf._schema[1].type == fastparquet.parquet_thrift.Type.INT96
>       out = dd.read_parquet(fn, engine="fastparquet", index=False).compute()

dask/dataframe/io/tests/test_parquet.py:1883: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
dask/base.py:342: in compute
    (result,) = compute(self, traverse=False, **kwargs)
dask/base.py:628: in compute
    results = schedule(dsk, keys, **kwargs)
dask/dataframe/io/parquet/core.py:96: in __call__
    return read_parquet_part(
dask/dataframe/io/parquet/core.py:654: in read_parquet_part
    dfs = [
dask/dataframe/io/parquet/core.py:655: in <listcomp>
    func(
dask/dataframe/io/parquet/fastparquet.py:1075: in read_partition
    return cls.pf_to_pandas(
dask/dataframe/io/parquet/fastparquet.py:1115: in pf_to_pandas
    df, views = pf.pre_allocate(size, columns, categories, index)
/usr/share/miniconda3/envs/test-environment/lib/python3.10/site-packages/fastparquet/api.py:797: in pre_allocate
    df, arrs = _pre_allocate(size, columns, categories, index, cats,
/usr/share/miniconda3/envs/test-environment/lib/python3.10/site-packages/fastparquet/api.py:1051: in _pre_allocate
    df, views = dataframe.empty(dtypes, size, cols=cols, index_names=index,
/usr/share/miniconda3/envs/test-environment/lib/python3.10/site-packages/fastparquet/dataframe.py:202: in empty
    values = type(bvalues)._from_sequence(values, copy=False, dtype=bvalues.dtype)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   ValueError: Buffer has wrong number of dimensions (expected 1, got 2)

pandas/_libs/tslibs/tzconversion.pyx:187: ValueError
```

Regression due to https://github.com/dask/fastparquet/pull/893 @jbrockmendel

--- a/.github/workflows/wheel.yml
+++ b/.github/workflows/wheel.yml
@@ -18,7 +18,7 @@ jobs:
             linux_archs: "aarch64"
             skip: "pp* *-musllinux_aarch64"
 
-    name: Python ${{ matrix.os }}
+    name: ${{ matrix.os }} ${{ matrix.architecture }} ${{ matrix.linux_archs }}
     env:
       CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: "delvewheel repair -w {dest_dir} {wheel}"
       CIBW_BEFORE_ALL: "pip install numpy cython"
@@ -71,9 +71,9 @@ jobs:
           # https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
           - os: ubuntu-22.04
             linux_archs: "i686"
-            skip: "pp*"
+            skip: "pp* *312*"
 
-    name: Python ${{ matrix.os }}
+    name: ${{ matrix.os }} ${{ matrix.architecture }} ${{ matrix.linux_archs }}
     env:
       CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: "delvewheel repair -w {dest_dir} {wheel}"
       CIBW_BEFORE_ALL: "pip install numpy cython"
@@ -128,7 +128,7 @@ jobs:
             linux_archs: "x86_64"
             skip: "pp*"
 
-    name: Python ${{ matrix.os }}
+    name: ${{ matrix.os }} ${{ matrix.architecture}}
     env:
       CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: "delvewheel repair -w {dest_dir} {wheel}"
       CIBW_BEFORE_ALL: "pip install numpy cython"
@@ -175,14 +175,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2019]
+        os: [windows-latest]
         architecture: ['x64']
         include:
           # https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
-          - os: windows-2019
+          - os: windows-latest
             skip: "*2*win* *win32 pp*"
 
-    name: Python ${{ matrix.os }}
+    name: ${{ matrix.os }} ${{ matrix.architecture}}
     env:
       CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: "delvewheel repair -w {dest_dir} {wheel}"
       CIBW_BEFORE_ALL: "pip install numpy cython"
@@ -233,7 +233,7 @@ jobs:
         os: [macos-latest]
         architecture: ["universal2",  "arm64"]
 
-    name: Python ${{ matrix.os }}
+    name: ${{ matrix.os }} ${{ matrix.architecture}}
     env:
       CIBW_BEFORE_ALL: "pip install numpy cython"
       CIBW_SKIP: ${{ matrix.skip }}

--- a/fastparquet/dataframe.py
+++ b/fastparquet/dataframe.py
@@ -206,7 +206,7 @@ def empty(types, size, cats=None, cols=None, index_types=None, index_names=None,
                 if bvalues.dtype.kind == "m":
                     dt = "m8[ns]" if PANDAS_VERSION.major < 2 else bvalues.dtype
                     values = np.zeros(shape=shape, dtype=dt)
-                    values = type(bvalues)._from_sequence(values, copy=False)
+                    values = type(bvalues)._from_sequence(values.view("int64"), copy=False, dtype=bvalues.dtype)
                 elif bvalues.dtype.kind == "M":
                     dt = "M8[ns]" if PANDAS_VERSION.major < 2 else bvalues.dtype
                     values = np.zeros(shape=shape, dtype=dt)

